<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1523</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="uiFPTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1121</width>
     <height>786</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="uiFPTypeCombo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>246</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fingerprint Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="uiFPLabel">
   <property name="geometry">
    <rect>
     <x>1155</x>
     <y>505</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>uiFPTypeCombo</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>change_fp_type()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiFPTable</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Form</receiver>
   <slot>depict_fp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>change_fp_type()</slot>
  <slot>depict_fp()</slot>
 </slots>
</ui>
